pacman/operations/router_algorithms/basic_dijkstra_routing.py

```diff
@@ -90,14 +93,18 @@ def __call__(self, placements, machine, machine_graph,
         self._update_all_weights(nodes_info)
 
         # each vertex represents a core in the board
-        progress = ProgressBar(
-            placements.n_placements, "Creating routing entries")
-        for placement in progress.over(placements.placements):
+        progress = None
+        if use_progress_bar:
```

Comment:
I think I ought to make a dummy progress bar instead which just passes things through.

Reply:
possibly, it does open the question about whom should build the progress bar. Given this multi use issue.

Reply:
SpiNNakerManchester/SpiNNUtils#53 will let this be cleaner